app/templates/courses/enrollments-view.jade

```diff
@@ -100,10 +100,16 @@ block content
 mixin prepaidCard(prepaid, className)
   .prepaid-card.bg-navy.text-center.p-a-2.rtl-allowed(class=className)
     h1.m-t-2.m-b-0= prepaid.openSpots()
+    - var includedCourseIDs = prepaid.get('includedCourseIDs')
     if prepaid.get('type') === 'starter_license'
       div.rtl-allowed(data-i18n="teacher.starter_licenses")
+    else if includedCourseIDs
+      div.rtl-allowed(data-i18n="teacher.customized_license")
     else
-      div.rtl-allowed(data-i18n="teacher.licenses_remaining")
+      div.rtl-allowed(data-i18n="teacher.full_license")
+    div.includedCourses
+      each id in (includedCourseIDs || [])
+        i.span=view.utils.courseAcronyms[id]
```

Comment:
Confusing to have `i` tag with class `span`. Do we need the `span` class, as opposed to just referencing the `i` tag in our style? If so, can we rename to something more descriptive, like `i.course-acronym`?